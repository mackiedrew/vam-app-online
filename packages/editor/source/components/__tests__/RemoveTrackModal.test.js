import Subject from "../RemoveTrackModal";

const testProps = {
  fileName: "File Name.wav",
  removeTrack: jest.fn(),
  clearModal: jest.fn()
};

describe("<Modal />", () => {
  it("renders without crashing", () => {
    shallow(<Subject {...testProps} />);
  });

  it("renders correctly", () => {
    const tree = renderer.create(<Subject {...testProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
