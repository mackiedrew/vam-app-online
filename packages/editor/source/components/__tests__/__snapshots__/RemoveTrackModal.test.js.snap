// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> renders correctly 1`] = `
<div
  className="modal"
>
  <div
    className="dialog"
  >
    <h2
      className="title"
    >
      Are you sure you want to remove File Name.wav?
    </h2>
    <p
      className="text"
    >
      Once it is removed it cannot be undone.
    </p>
    <button
      className="yes-button"
      onClick={[Function]}
    >
      Yes
    </button>
    <button
      className="no-button"
      onClick={[Function]}
    >
      No
    </button>
  </div>
</div>
`;
